examples/demo/gloo/molecular_viewer.py

```diff
@@ -74,7 +74,7 @@
     pos.z += v_radius*z;
     vec3 pos2 = pos.xyz;
     pos = u_projection * pos;
-    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
+//    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
```

Comment:
I was actually referring to [the third case](https://github.com/vispy/vispy/blob/master/vispy/visuals/volume.py), but maybe it's not used in WebGL.